examples/compare_CPMG_echo.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from pyspecdata import find_file, figlist_var
+from pyspecProcScripts import lookup_table, select_pathway
+
+with figlist_var() as fl:
+    for thisfile, exp_type, nodename, complex_cpmg, thislabel in [
+        #(
+        #    "240702_13p5mM_TEMPOL_CPMG.h5",
+        #    "ODNP_NMR_comp/CPMG",
+        #    "CPMG_13",
+        #    False,
+        #    "CPMG simple cyc",
+        #),
+        #(
+        #    "240702_13p5mM_TEMPOL_CPMG.h5",
+        #    "ODNP_NMR_comp/CPMG",
+        #    "CPMG_17",
+        #    True,
+        #    "CPMG large cyc SW = 2.0",
+        #),
+        (
+            "240702_13p5mM_TEMPOL_CPMG.h5",
+            "ODNP_NMR_comp/CPMG",
+            "CPMG_19",
+            True,
+            "CPMG large cyc SW = 10",
+        ),
+        #(
+        #    "240702_13p5mM_TEMPOL_echo.h5",
+        #    "ODNP_NMR_comp/Echoes",
+        #    "echo_10",
+        #    False,
+        #    "echo large cyc SW = 2.0",
+        #),
+        (
+            "240702_13p5mM_TEMPOL_echo.h5",
+            "ODNP_NMR_comp/Echoes",
+            "echo_12",
+            False,
+            "echo large cyc SW = 10.0",
+        ),
+    ]:
+        thisd = find_file(
+            thisfile, exp_type=exp_type, expno=nodename, lookup=lookup_table
+        )
+        thisd.squeeze()
+        fl.next("raw data for %s" % thislabel)
+        rows = np.prod([thisd.shape[j] for j in thisd.dimlabels[:-1]])
+        if rows < 500:
+            fl.image(thisd)
+        else:
+            fl.image(thisd, interpolation="bilinear")
```

Comment:
I just updated pyspecdata -- it should be possible to replace all of this with `fl.image(thisd, interpolation='auto')`